testing/tpm2-tss/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor:
+# Maintainer:
+pkgname="tpm2-tss"
+pkgver="2.3.1"
+pkgrel=0
+pkgdesc="TPM2 tss library"
+url="https://github.com/tpm2-software/tpm2-tss/"
+arch="all"
+license="BSD-2-Clause"
+makedepends="openssl-dev doxygen linux-headers cmocka-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://github.com/tpm2-software/tpm2-tss/releases/download/${pkgver}/tpm2-tss-${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--enable-unit \
```

Comment:
If you're enabling unit checks it would make sense to actually run them in `check()`

Reply:
i disabled building them for now... we have to fix them upstream first.